### PR TITLE
add pylammpsmpi

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@srmnitc Can I add you as a maintainer?

**Comment by srmnitc** [comment]
> @srmnitc Can I add you as a maintainer?

Yes sure!

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/pylammpsmpi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  skip: true  # [win | py2k]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - mpi4py
+    - openmpi
```
Comment (scopatz) [inline recipes/pylammpsmpi/meta.yaml]: Please see the documentation here for MPI: https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi

Comment (jan-janssen) [reply recipes/pylammpsmpi/meta.yaml]: But this only works with open mpi, because mpich does not support communicating via stdin.

Comment (scopatz) [reply recipes/pylammpsmpi/meta.yaml]: Then only inlcude openmpi in `conda_build_config.yaml`

Comment (jan-janssen) [reply recipes/pylammpsmpi/meta.yaml]: Done

**Comment by jan-janssen** [comment]
the dependencies are not available on Windows but in general the package itself is operation system independent and therefore I switched to a noarch recipe.

**Pull request merged**